tests/ProjectCodeTest.php

```diff
@@ -81,6 +83,15 @@ function (\ReflectionClass $interface) {
             'PhpCsFixer\Config' => array('create'),
             'PhpCsFixer\Fixer\FunctionNotation\MethodArgumentSpaceFixer' => array('fixSpace'),
             'PhpCsFixer\Fixer\Import\OrderedImportsFixer' => array('sortingCallBack'),
+            'PhpCsFixer\FixerConfiguration\FixerConfigurationResolver' => array('addOption', 'mapRootConfigurationTo'),
+            'PhpCsFixer\FixerConfiguration\FixerOption' => array(
+                'removeDefault',
```

Comment:
This was a method I wrote and then removed, but I forgot to remove it from here too.